moralization/tests/test_transformers_model_manager.py

```diff
@@ -9,6 +10,15 @@ def gen_instance(tmp_path):
     return TransformersModelManager(tmp_path)
 
 
+@pytest.fixture
+def gen_instance_dm(data_dir):
+    dm = DataManager(data_dir)
+    dm.docdict_to_lists()
+    dm.lists_to_df()
```

Comment:
could these two `docdict_to_lists` and `lists_to_df` methods be called automatically as part of the DataManger init, or are there cases where you wouldn't want to do this?